webapp/src/components/modals/channel_settings/channel_settings_filter.tsx

```diff
@@ -85,6 +85,39 @@ export default class ChannelSettingsFilter extends React.PureComponent<ChannelSe
         return null;
     };
 
+    renderInclusionDropdownOption = (data: {value: string; label: string}, meta: {context: string}): JSX.Element | string => {
+        const {value, label} = data;
+        const {context} = meta;
+
+        // context === value means it is rendering the selected value
+        if (context === 'value') {
+            return label;
+        }
+
+        // otherwise it is rendering an option in the open dropdown
+        let subtext = '';
+        switch (value) {
+        case FilterFieldInclusion.INCLUDE_ANY:
+            subtext = 'Includes either of the values (or)';
+            break;
+        case FilterFieldInclusion.INCLUDE_ALL:
+            subtext = 'Includes all of the values (and)';
+            break;
+        case FilterFieldInclusion.EXCLUDE_ANY:
+            subtext = 'Excludes all of the values';
```

Comment:
`EXCLUDE_ANY` means that if the incoming webhook contains any of the selected values, the post will not be made. It is currently named `ANY` because we check if `any` of the values exist. But as far UX wording, we excluding `all` of the values.